Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -137,6 +138,8 @@ def __init__(self, struct_or_class, cf_type, namespace, name, parent_class_name,
                         self.custom_encoder = True
                     elif attribute == 'SupportWKKeyedCoder':
                         self.support_wkkeyedcoder = True
+                    elif attribute == 'DebugDecodingFailure':
```

Comment:
Could you add this attribute to a structure in Source/WebKit/Scripts/webkit/tests/TestSerializedType.serialization.in and run make -C Source/WebKit/Scripts/webkit/tests and include those changes in this PR?  We are trying to keep a complete test suite of the attributes and generator and its expected output.